src/app/route/CategoryPage/CategoryPage.component.js

```diff
@@ -340,15 +451,16 @@ class CategoryPage extends Component {
     }
 
     renderItemCount() {
-        const { items, totalItems, isLoading } = this.props;
+        const { totalItems, isInfoLoading, isPagesLoading } = this.props;
+        const { productsLoaded } = this.getPageParams();
 
         return (
             <p block="CategoryPage" elem="ItemsCount">
-                {isLoading
+                {isInfoLoading
```

Comment:
Add spaces between {}

Reply:
I believe we can describe it here: https://github.com/scandipwa/base-theme/blob/master/.eslintrc#L46 By putting { "children": true } as per https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md

Reply:
It also warns then about other ugly parts as well (like useless new line curly bracket)

Reply:
We have the one configured. Can you please check if you have eslint installed as an extension for VSCode?

Reply:
Sure we have, I just send you link to the actual configuration file we are using now. I mean to make this working I had to change it from `"react/jsx-curly-spacing": [2, {"when": "always", "allowMultiline": false} ],` to `"react/jsx-curly-spacing": [2, {"when": "always", "allowMultiline": false, "children": true} ],`, so that way it will also check it for children components (bit that we write inside `<tag>{codeToCheck}</tag>`).